cool-core/src/main/java/com/nus/cool/loader/DataLoader.java

```diff
@@ -74,9 +72,12 @@ public static Builder builder(String dataSetName,
      */
     public void load() throws IOException {
         writer.Initialize();
-      while (reader.hasNext()) {
-        writer.Add(parser.parse(reader.next()));
-      }
+        // read the csv column
```

Comment:
This would break other reader. the reader is not expecting a header in other file formats, nor do those formats have. I suggest, if you want this feature, remove the csv header and possibly check the schema during initialization (ex, in CSVDataLoaderConfig::createTupleReader) not here.

Reply:
agree